src/main/java/org/veupathdb/service/eda/ds/plugin/pass/LineplotPlugin.java

```diff
@@ -51,6 +51,7 @@ public ConstraintSpec getConstraintSpec() {
       .dependencyOrder("yAxisVariable", "xAxisVariable", "overlayVariable", "facetVariable")
       .pattern()
         .element("yAxisVariable")
+          .minValues(2)
```

Comment:
the ex they gave was def for the proportion mode.. i just wasnt sure how generally i should interpret this. is it any more reasonable to have a cont var on the yaxis that only has 1 value than a categorical one?

Reply:
For continuous vars i think 1 unique value is reasonable but 1 data point is not so reasonable (now i'm realizing i wasn't sure if 'value' meant unique value or data point). The only pause i have about a var constraint against 1 data point is that in scatter we can plot 1 point. Seems a reasonable thing that x axis should also have the same constraint as the y if we do apply this to all var types in the line?

Reply:
any summary value we provide for a single value (not data point) would just be that value... but i guess maybe thats fine. similar for the xaxis, itd be weird to have a line plot where there was only one xaxis tick, since thats not really a line at all. but maybe its better to let the user do it and realize whats happened than to constrain against it? i dont think we constrain against strata vars w a single value so.... yea i guess well go w your multiple patterns thing

Reply:
... ugh. or does that also suggest itd be fine to calculate proportions that were all '1' as well if there was a single value?

Reply:
maybe we can ask at the next dataviz meeting if constraining against variables w a single value is something we want to do generally or not